### ISSUE TITLE
JSON Query for property names with non-alphanumeric characters

### ISSUE THREAD

**Issue by ngiessel** [rangav/thunder-client-support#348]
Hi,

I'm attempting to create a test that saves a json property value to an environment variable and couldn't figure out how to query the value of non-alphanumeric properties (such as "@odata.etag"). It throws an "Error parsing json query".

How would you go about this? Or is this even possible?

**Comment by rangav** [comment]
Hi @ngiessel thanks for reporting the issue, can you share sample json, I will test and verify.

**Comment by ngiessel** [comment]
Sure. The `loginId` property works fine.
```json
{
  "@odata.etag": "W/\"JzQ0O3EwcnRsa0hmL3B0SUg1eWVnWUxLVHZyOUowMHhlUk5xRWpRU3FEZ0RObVE9MTswMDsn\"",
  "loginId": "user@example.com",
}
```

**Comment by rangav** [comment]
Thanks for sharing, I will check and get back to you

**Comment by ngiessel** [comment]
I forgot to mention the "Error parsing json query" error didn't occur when using the Set Env Variable test, but while trying to verify the query string was valid by using a Json Query-contains test.

**Comment by MathieuDR** [comment]
I have the same problem with the following response for examtle

```json
{
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
  "title": "One or more validation errors occurred.",
  "status": 400,
  "traceId": "00-aa252c4a3ea9814a89ca065db791b079-d1dff01358f86e90-00",
  "errors": {
    "Platform.Id": [
      "The Id field is required."
    ]
  }
}
```

When I try to get `json.errors["Platform.Id"][0]` I receive "Error parsing json query". Same if I try with a single quote, escaping the quotles or period.

**Comment by rangav** [comment]
This bug is now fixed in v1.12.0

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.12.0

please let me know if any feedback
